.github/workflows/ci.yml

```diff
@@ -127,8 +127,6 @@ jobs:
     strategy:
       matrix:
         execution: [BaseOnPython]
-    env:
-      MODIN_TEST_DATASET_SIZE: "small"
```

Comment:
Due to the different data sizes, this execution may break when others work fine. Sometimes when writing tests we rely on a certain number of columns. As an example, a test with a pivot.

Reply:
There is also an option to increase the number of columns a little for the smallest data set.